src/middlewares/event-dispatcher.js

```diff
@@ -135,6 +135,10 @@ function onClickableComponentsHandler(store: any, action: Object, player: Player
     case 'Volume':
       onVolumeClicked(store, action, player);
       break;
+
+    case 'Stereo':
+      player.dispatchEvent(new FakeEvent(FakeEvent.Type.USER_CLICKED_STEREO_MODE));
```

Comment:
Maybe add a payload which indicates whether the stereo is on or off?

Reply:
second @dan-ziv 's comment, also change to  `USER_CLICKED_VR_STEREO_MODE`